### DISCUSSION TITLE
How to make the text scrollable?

### DISCUSSION THREAD

**Discussion by softshipper** [mdx-editor/editor#617] [Q&A]
Hi all,
Could someone help make the textarea scrollable like this example

![image](https://github.com/user-attachments/assets/8ee546b2-fe69-4834-9098-4464a19f68f4)

The example was taken from https://mdxeditor.dev/.

Best regards

**Comment by ochnios** [comment]
Here is the code for the example: https://github.com/mdx-editor/site/blob/master/app/HomepageEditor.tsx 
You just need to define the `overflow-y: auto` and `max-height` attributes of the container in which MDXEditor is placed.
